eslint/babel-eslint-tests/test/integration/eslint/verify.js

```diff
@@ -1794,6 +1803,31 @@ describe("verify", () => {
           { "no-unreachable": 1 },
         );
       });
+
+      it("should not be unamed", () => {
```

Comment:
Typo here:

```suggestion
      it("should not be unnamed", () => {
```
Or maybe the following message for consistency?

```suggestion
      it("should work with func-names", () => {
```